src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/TestBase.cs

```diff
@@ -72,7 +73,7 @@ public abstract partial class TestBase : IAsyncLifetime
     /// <summary>
     ///  An <see cref="IRazorLogger"/> for the currently running test.
     /// </summary>
-    protected IRazorLogger Logger => _logger ??= new LoggerAdapter(new[] { LoggerFactory.CreateLogger(GetType()) }, new TelemetryReporter(LoggerFactory));
```

Comment:
I'm not entirely sure, but I wouldn't be surprised if this means we were reporting telemetry from tests, which is kinda funny

Reply:
we likely were, at least release build tests. Woops 🤦‍♀️